atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/impl/expectations/TrackingRowColumnRangeIteratorTest.java

```diff
@@ -57,16 +60,15 @@ public long applyAsLong(Entry<Cell, Value> value) {
 
     @Test
     public void oneElementTrackingRowColumnRangeIteratorIsWiredCorrectly() {
-        Consumer<Long> tracker = spy(TrackingIteratorTestUtils.noOp());
+        Consumer<Long> tracker = spy(NO_OP);
```

Comment:
nit / non actionable: I don't normally like using constants if there is only one thing, though I'm fine either way